packages/shared/src/lib/core/network/stores/networks.store.ts

```diff
@@ -11,15 +12,25 @@ export const networks: Writable<Network[]> = writable([])
 
 export function initializeNetworks(): void {
     const profile = get(activeProfile)
+    let _networks: Network[] = []
     if (profile?.network) {
         const stardustNetwork = new StardustNetwork(profile.network)
         const chains = profile.network.chainConfigurations
             .map((chainConfiguration) => {
                 return new IscChain(chainConfiguration)
             })
             .filter(Boolean) as IEvmNetwork[]
-        networks.set([stardustNetwork, ...chains])
+        _networks = [stardustNetwork, ...chains]
     }
+
+    if (features.network.evmNetworks.enabled) {
+        profile.evmNetworks?.forEach((evmNetwork) => {
+            const newNetwork = new EvmNetwork(evmNetwork)
+            _networks.push(newNetwork)
```

Comment:
i would just make it one line:

```suggestion
            _networks.push(new EvmNetwork(evmNetwork)
)
```